From: =?utf-8?q?R=C3=A9mi_Vanicat?= <vanicat@debian.org>
Date: Sun, 23 Aug 2015 10:20:17 +0200
Subject: Do not build .elc for magit elpa package

---
 Makefile | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7d91cd5..97d1c32 100644
--- a/Makefile
+++ b/Makefile
@@ -56,6 +56,12 @@ lisp:
 	@$(RM) $(ELCS) $(ELGS) # temporary cleanup kludge
 	@$(MAKE) -C lisp lisp
 
+lisp/magit-version.el:
+	@$(MAKE) -C lisp magit-version.el
+
+lisp/magit-autoloads.el:
+	@$(MAKE) -C lisp magit-autoloads.el
+
 docs:
 	@$(MAKE) -C Documentation all
 
@@ -216,7 +222,7 @@ define magit_pkg
 endef
 # '
 export magit_pkg
-magit-$(VERSION).tar: lisp info
+magit-$(VERSION).tar: info lisp/magit-version.el lisp/magit-autoloads.el
 	@printf "Packing $@\n"
 	@$(MKDIR) magit-$(VERSION)
 	@printf "$$magit_pkg\n" > magit-$(VERSION)/magit-pkg.el
