From: =?utf-8?q?R=C3=A9mi_Vanicat?= <vanicat@debian.org>
Date: Sun, 23 Aug 2015 10:20:17 +0200
Subject: Do not build .elc for magit elpa package

---
 Makefile | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile b/Makefile
index afa85df..79ddcea 100644
--- a/Makefile
+++ b/Makefile
@@ -54,6 +54,12 @@ lisp:
 	@$(RM) $(ELCS) $(ELGS) # temporary cleanup kludge
 	@$(MAKE) -C lisp lisp
 
+lisp/magit-version.el:
+	@$(MAKE) -C lisp magit-version.el
+
+lisp/magit-autoloads.el:
+	@$(MAKE) -C lisp magit-autoloads.el
+
 docs:
 	@$(MAKE) -C Documentation all
 
