From: =?utf-8?q?R=C3=A9mi_Vanicat?= <vanicat@debian.org>
Date: Mon, 6 Jul 2020 15:41:00 +0200
Subject: Update version and require of git-commit

---
 lisp/git-commit.el | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lisp/git-commit.el b/lisp/git-commit.el
index 4ed2072..063601e 100644
--- a/lisp/git-commit.el
+++ b/lisp/git-commit.el
@@ -11,9 +11,10 @@
 ;;	Marius Vollmer <marius.vollmer@gmail.com>
 ;; Maintainer: Jonas Bernoulli <jonas@bernoul.li>
 
-;; Package-Requires: ((emacs "25.1") (dash "20180910") (transient "20190812") (with-editor "20181103"))
+;; Package-Requires: ((emacs "25.1") (dash "2.17.0") (transient "0.2.0") (with-editor "2.9.1"))
 ;; Keywords: git tools vc
 ;; Homepage: https://github.com/magit/magit
+;; Version: 2.99.0
 
 ;; This file is not part of GNU Emacs.
 
